docs/content/modules/tensor_mechanics/introduction/StressDivergence.md

```diff
@@ -0,0 +1,34 @@
+#Stress Divergence Kernel Settings
+
+Within the tensor mechanics module, we have three separate ways to calculate the strain and stress:
+
+1. Linearized elasticity total small strain problems: use [ComputeSmallStrain](auto::/Materials/ComputeSmallStrain) and [ComputeLinearElasticStress](auto::/Materials/ComputeLinearElasticStress)
+2. Linearized elasticity incremental small strain: use [ComputeIncrementalSmallStrain](auto::/Materials/ComputeIncrementalSmallStrain) and [ComputeFiniteStrainElasticStress](auto::/Materials/ComputeFiniteStrainElasticStress)
+3. Large deformation problems, including elasticity and/or plasticity: use [ComputeFiniteStrain](auto::/Materials/ComputeFiniteStrain) and [ComputeFiniteStrainElasticStress](auto::/Materials/ComputeFiniteStrainElasticStress)
+
+The linearized elasticity problems are calculated on the reference mesh.  In the linearized elasticity total small strain formulation, [ComputeSmallStrain](auto::/Materials/ComputeSmallStrain) a rotation increment is not used; in [ComputeIncrementalSmallStrain](auto::/Materials/ComputeIncrementalSmallStrain) the rotation increment is defined as the identity tensor.  Both the total small strain and the incremental small strain classes pass to the stress divergence kernel a stress calculated on the reference mesh ( $ \sigma(X) $ ).
+
+In the third set of the plug-and-play tensor mechanics classes, the large deformation formulation calculates the strain and stress on the deformed (current) mesh.  As an example, at the end of the [ComputeFiniteStrain](auto::/Materials/ComputeFiniteStrain) class the strains are rotated to the deformed mesh, and in the [ComputeFiniteStrainElasticStress](auto::/Materials/ComputeFiniteStrainElasticStress) class the stress is rotated to the deformed mesh.  Newer material models, such as crystal plasticity models and creep models, also rotate the strain and stress to the deformed mesh.  In these large deformation classes, the stress passed to the stress divergence kernel is calculated with respect to the deformed mesh ( $ \sigma(x) $ ).
+
+As users and developers, we must take care to ensure consistency in the mesh used to calculate the strain and stress and the mesh used to calculate the residual from the stress divergence equation.  In the [StressDivergenceTensors](auto::/Kernels/StressDivergenceTensors) kernel, or the various stress divergence actions, the parameter `use_displaced_mesh` is used to determine if the deformed or the reference mesh should be used:
```

Comment:
@aeslaughter In this line I would like to link to the StressDivergenceTensors.md file, but the webpage links to StressDivergenceTensorsTruss.md instead--is there a way to search for only the provided string instead of files that contain the provided string?